src/components/Phoneicon/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import PropTypes from "prop-types";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faPhone } from "@fortawesome/free-solid-svg-icons";
+
+library.add(faPhone);
+export const PhoneEmoji = styled(FontAwesomeIcon)`
+  color: gray;
+  font-size: 24px;
+  ${({ variant }) =>
+    variant === "secondary" &&
+    css`
+      color: blue;
+    `}
+`;
+export const CallEmoji = ({ variant }) => (
+  <PhoneEmoji icon={faPhone} variant={variant} />
+);
+CallEmoji.propTypes = {
+  // eslint-disable-next-line react/require-default-props
```

Comment:
in general is never a good practice to skip eslint checks. Sometimes it is useful, but most of the time those checks are in place to improve code quality. In this case, the solution to your issue would be to provide defaultProps as well. Please refer to the doc about propTypes that I linked above, if not clear, slack me 😉